bpo-29521 Fix two minor documentation build warnings

JDLH opened the pull request:
Much of bpo-29521 was fixed in parallel with commit 
e7ffb99 .  This cleans up the rest.

Apply parallel change to Doc/make.bat to read 
"set SPHINXOPTS=-D latex_elements.papersize="
I don't have a Windows system on which to observe the warning,
but it should be necessary.

The warning:

.../workspace/cpython_github/Doc/faq/windows.rst:303: 
    WARNING: unknown option: -t

In the Windows FAQ, How do I keep editors from inserting tabs 
into my Python source?, contained a reference to a Python -t
option. In Python 2.x, this caused Python to issue warnings
about lines with mixed spaces and tabs, but as of Python 3.6
it does nothing. 
 
Per discussion at http://bugs.python.org/issue29387, take
their wording. Python [3] raises an IndentationError or 
TabError. Tabnanny is now a module.

nicolas17 commented:
There is some irony in contributing this "we don't accept pull requests" template as a pull request. Shouldn't it be sent in the issue tracker? :sweat_smile:

brettcannon commented:
This was accepted through hg.python.org/cpython

willingc commented:
Thanks @brettcannon

Commit:
bpo-29521 Fix two minor documentation build warnings

Much of bpo-29521 was fixed in parallel with commit 
e7ffb99 .  This cleans up the rest.

Apply parallel change to Doc/make.bat to read 
"set SPHINXOPTS=-D latex_elements.papersize="
I don't have a Windows system on which to observe the warning,
but it should be necessary.

The warning:

.../workspace/cpython_github/Doc/faq/windows.rst:303: 
    WARNING: unknown option: -t

In the Windows FAQ, How do I keep editors from inserting tabs 
into my Python source?, contained a reference to a Python -t
option. In Python 2.x, this caused Python to issue warnings
about lines with mixed spaces and tabs, but as of Python 3.6
it does nothing. 
 
Per discussion at http://bugs.python.org/issue29387, take
their wording. Python [3] raises an IndentationError or 
TabError. Tabnanny is now a module.

briancurtin reviewed (approved):
Looks good, thanks for submitting!
